lib/cartopy/tests/mpl/test_images.py

```diff
@@ -150,6 +152,21 @@ def test_imshow_wrapping():
     assert ax.get_xlim() == (-180, 180)
 
 
+def test_imshow_rgba():
+    # tests that the alpha of a RGBA array passed to imshow is set to 0
+    # instead of masked
+    z = np.random.rand(100, 100)
```

Comment:
Do we really need random data? At least add a seed.

Reply:
Nope